pkg/kubelet/kubelet.go

```diff
@@ -2388,13 +2388,17 @@ func (kl *Kubelet) isOutOfDisk() bool {
 	// Check disk space once globally and reject or accept all new pods.
 	withinBounds, err := kl.diskSpaceManager.IsRuntimeDiskSpaceAvailable()
 	// Assume enough space in case of errors.
-	if err == nil && !withinBounds {
+	if err != nil {
+		glog.Errorf("failed to check if disk space is available for the runtime: %v", err)
```

Comment:
I am not sure that is a good idea or not. We do have customers using different graph drivers other than devicemapper, aufs, and overlayfs. The current approach we are having is that we recommend the user to use the support graph drivers since we have proper supports for those drivers, but we cannot force the user to use them. 

I do agree that if kubelet failed to check the disk usages, Kubelet should error out that loudly. Just my 2 cents.